### DISCUSSION TITLE
Save as HTML site

### DISCUSSION THREAD

**Discussion by abrahampe** [dedoc/scramble#170] [Ideas]
Would be nice to be able to save the entire generated docs as an pure html/css web site. Is it possible? Is there anybody working on it?

**Comment by romalytvynenko** [comment]
Hey @abrahampe 

Do you mind sharing why you need this?

**Reply by abrahampe** [reply]
I would like to host it out of my main application (together with others docs, I already have there) , so it would be avaliable to coders without the need of installing all dev environment, and I would not like to enable it in production too, at least, most of docs generators has this feature. But It is my personal need, I understand if most don't share same need.
